content/community/ospp/2022-projects.md

```diff
@@ -79,21 +79,22 @@ AOSC's package information site is now severely disconnected from the current AO
   - https://github.com/AOSC-Dev/packages-site
 - License: GNU Public License Version 2
 
-# Add Missing Documents on AOSC Wiki and Update Existing Documents
+# Add Missing Documents and Update Existing Documents on AOSC Wiki
 
 There are many AOSC projects missing documents on AOSC Wiki, and many existing documents are outdated. This project will update existing documents and add undocumented project documents to AOSC Wiki.
 
-- Difficulty: Advanced [Temporary]
+- Difficulty: Advanced [Pending]
 - Mentor: Kaiyang Wu
 - Mentor contact: origincode@aosc.io
 - Project Requirements:
-  - Add documents for the projects in the work ticket, make sure the English contents are the latest if possible
+  - Communicate with community contributors and collect entries for repair or writing, and develop a work plan.
```

Comment:
- "entries" => "Wiki entries"
- "For improve to writing" => "to repair or create,"
- "develop" => "draft"